tests/system/python/helpers/plugin_and_service.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think, We recently added a utils module for system test. Can we please use that? /cc @Singhal-Vaibhav

Reply:
This is under helpers/ and we can’t place everything in utils.py.

Reply:
I suggest to defer the decision adding a new helper; please use whatever you need in the test (even if its violating the DRY principle) and we refactor it after rest api system tests are done!